src/main/scripts/src/app/core/header/header.component.scss

```diff
@@ -97,6 +102,7 @@
 
   ul {
     box-shadow: $default-shadow;
+    background: white;
```

Comment:
Please use `$secondary-color` - which is translated to `#ffffff` instead of just `white` color.